subsystem-arm/implementations/hand/hand-modes.hpp

```diff
@@ -39,6 +40,17 @@ namespace sjsu::arm
             return arguments_;
         }
 
+        static hand_arguments ManualHand(hand_arguments arguments_)
+        {
+            hand_arguments temp_;
+            temp_.pinky_angle = arguments_.pinky_angle;
+            temp_.ring_ange = arguments_.ring_angle;
+            temp_.middle_angle = arguments_.middle_angle;
+            temp_.index_angle; = arguments_.index_angle;
+            temp_.thumb_angle = arguments_.thumb_angle;
```

Comment:
Good job but for this more all you would have to do is return the thing you passed in, as long as all the conversions are handled somewhere else we should be okay.